requirements.txt

```diff
@@ -1,4 +1,4 @@
-qiskit-terra>=0.18.0
+qiskit-terra>=0.18.0,<0.20.0
```

Comment:
Would be good to leave a comment/todo that explains why the provider does not work with the latest version of terra. Guess it's a temporary workaround only?

Reply:
Sorry, wasn't super clear about it - left a comment [here](https://github.com/Qiskit/qiskit-ibm-provider/pull/306#issuecomment-1086766863) 

the docs are failing after the update to terra 0.20.0, and I believe it should be a temporary workaround until the new aer release

Reply:
I merged https://github.com/Qiskit/qiskit-ibm-provider/pull/309 that includes this change. Make sure to remove it prior to the first release. LGTM